lib/collapsible.php

```
@@ -0,0 +1,113 @@
+        return $content;
+    }
+
+    $content = preg_replace(
+        '#<details\b[^>]*\bclass="[^"]*\bcollapsible-item\b[^"]*"[^>]*>\s*<summary>(.*?)</summary>#is',
+        '<h4>$1</h4>',
+        $content,
+        -1,
+        $count
+    );
+
+    return $count > 0 ? preg_replace('#</details>#i', '', $content) : $content;
```

PHPStan (2.2.8)

[warning] Function zw_collapsible_flatten() should return string but returns string|null.


(return.type)